scripts/asm-installer/install_asm

```diff
@@ -237,8 +238,12 @@ main() {
     exit_if_not_cluster_admin
   fi
 
-  if should_validate && ! is_managed; then
-    ensure_istio_namespace_exists
+  if ! is_managed; then
```

Comment:
Added this check to be consistent with the logic before this PR and avoid creating the ns which might break MCP.